src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -157,71 +157,48 @@ ColumnLayout {
     }
 
     RowLayout {
```

Comment:
This whole thing should really be inside a loader, because right now we are instantiating this row for every single subscription management line item in the list. this is a bad pattern throughout this whole file

Reply:
Agree but would prefer to do any structural refactoring separately.